src/main/webapp/transcript.js

```diff
@@ -0,0 +1,71 @@
+
+const TRANSCRIPT_CONTAINER = 'transcript-container';
+
+/**
+ * Sends a POST request to the transcript.
+ */
+function sendPostToTranscript() {
+  // TODO: Update with user input.
+  const paramsString = 'id=123456789&video=3ymwOvzhwHs';
+  const params = new URLSearchParams(paramsString);
+  fetch('/transcript', {method: 'POST', body: params})
+      .then(fetchTranscriptLines(paramsString));
+}
+
+/**
+ * Fetches the transcript lines from \transcript.
+ *
+ * <p>{@code paramsString} indicates the video ID
+ * to fetch the transcript from.
+ */
+function fetchTranscriptLines(paramsString) {
+  fetch(
+      '/transcript' +
```

Comment:
Add const for /transcript and maybe ?

Reply:
Alternatively use UrlBuilder, but it looks like this is using hardcoded test param string?

Reply:
I think the question mark will be removed when I use the actual URL from your branch. 

I made an issue for converting to Url Buider later #42

Reply:
Sounds good, maybe add a TODO?

Reply:
Thanks for reminding me! Done